docs/react/install.en-US.md

```diff
@@ -31,6 +31,27 @@ $ npm install antd@beta --save
 
 > **Past releases**：https://github.com/ant-design/ant-design/releases
 
+## Import in Browser
+
+We provide `antd.js` `antd.css` and `antd.min.js` `antd.min.css` under `antd/dist` in antd's npm package, in order to import all the components of antd directly. Also, you can use [npmcdn](https://npmcdn.com/).
+
+> It's not a recommended way to use built file, for you cannot get bugfixes from the dependencies of antd.
```

Comment:
I am doing Node.js and React development, so I know enough about npm packages. :smile: 
I will update a little bit this statement to detail a little bit about the benefits of using NPMs.